core/domain/stats_jobs_continuous.py

```diff
@@ -103,6 +103,9 @@ def reduce(key, stringified_values):
                 <exploration_id>:<exploration_version>:<state_name>
             stringified_values: list(str). A list of stringified_values of the
                 submitted answers.
+
+        Yields:
+            Error.
```

Comment:
This isn't useful at all. When is it yielded? Under what circumstances? You will need to read and understand the method to find out.

Docstrings should always be written to help the person reading the code. If we start just writing them just for the sake of it then it's a bit pointless.

Reply:
Done.

Reply:
Btw, for this case, where is the "Error" that's being yielded? As far as I can tell all the yields are just str.

So the docstring should just be:

```
    Yields:
        str. One of the following strings:
        - ...
        - ...
```

Please check the others too. It's really not a great idea to introduce docstrings that have the wrong types since that's only going to confuse devs. If you are unsure about types, ask me.